helm-charts/binderhub/values.yaml

```diff
@@ -0,0 +1,560 @@
+binderhub:
+  jupyterhub:
+    #=== VALUES BELOW HERE ARE COPIED FROM BASEHUB VALUES AND SHOULD BE UPDATED ===#
+    #=== IF BASEHUB CHANGES ===#
+    custom:
+      2i2c:
+        # Should 2i2c engineering staff user IDs be injected to the admin_users
+        # configuration of the JupyterHub's authenticator by our custom
+        # jupyterhub_config.py snippet as declared in hub.extraConfig?
+        add_staff_user_ids_to_admin_users: false
+        add_staff_user_ids_of_type: ""
+        staff_github_ids:
+          - choldgraf
+          - consideRatio
+          - damianavila
+          - GeorgianaElena
+          - sgibson91
+          - yuvipanda
+        staff_google_ids:
+          - choldgraf@2i2c.org
+          - erik@2i2c.org
+          - damianavila@2i2c.org
+          - georgianaelena@2i2c.org
+          - sgibson@2i2c.org
+          - yuvipanda@2i2c.org
+    ingress:
+      enabled: true
+      annotations:
+        nginx.ingress.kubernetes.io/proxy-body-size: 256m
+        kubernetes.io/ingress.class: nginx
+        cert-manager.io/cluster-issuer: letsencrypt-prod
+    proxy:
+      service:
+        type: ClusterIP
+      chp:
+        nodeSelector:
+          hub.jupyter.org/node-purpose: core
+        resources:
+          requests:
+            # FIXME: We want no guarantees here!!!
+            # This is lowest possible value
+            cpu: 0.01
+            memory: 64Mi
+          limits:
+            memory: 1Gi
+      traefik:
+        image:
+          tag: v2.4.8
+        nodeSelector:
+          hub.jupyter.org/node-purpose: core
+        resources:
+          requests:
+            memory: 64Mi
+          limits:
+            memory: 1Gi
+    singleuser:
+      {{- if .Values.dask-gateway.enabled }}
+      # Almost everyone using dask by default wants JupyterLab
```

Comment:
Unfortunately we can't template values.yaml files :(

Reply:
![](https://media.giphy.com/media/8qCWiQ4a46oo73keKR/giphy.gif)